wagtail/test/dummy_external_storage.py

```diff
@@ -26,11 +27,16 @@ def _open(self, name, mode="rb"):
         # File object
         return DummyExternalStorageFile(open(self.wrapped.path(name), mode))
 
-    # Wrap all other functions
-
     def _save(self, name, content):
+        file_pos = content.tell()
+        if file_pos != 0:
```

Comment:
I know this is just a test util and we do not have tests for this class, plus there is not much value to test this, but if desired we can add the following test to cover this line.

```py


    def test_save_with_incorrect_file_object_position(self):
        # This is a 1x1 black png
        png = (
            b"\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00"
            b"\x00\x01\x00\x00\x00\x01\x08\x06\x00\x00\x00"
            b"\x1f\x15\xc4\x89\x00\x00\x00\rIDATx\x9cc````"
            b"\x00\x00\x00\x05\x00\x01\xa5\xf6E@\x00\x00"
            b"\x00\x00IEND\xaeB`\x82"
        )
        simple_png = SimpleUploadedFile(
            name="test.png", content=png, content_type="image/png"
        )
        simple_png.read()
        with self.assertRaisesMessage(
            ValueError,
            "Content file pointer should be at 0 - got 70 instead",
        ):
            DummyExternalStorage().save("test.png", simple_png)
```